src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -9,51 +9,114 @@
-              ) {
-                console.log('Invalid recaptcha');
-                return false;
-              } else {
-                form.submit();
-              }
-            },
-          });
-        } catch (e) {
-          grecaptcha.reset();
+  let checkEnv = window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b|\buat\b/);
+  let $feedbackForm = $('#qg-page-feedback-form');
+
+  let setUrlEnableCaptcha = () => {
+    // if environment is not PROD then use test submission handler link
+    checkEnv !== -1 ? $feedbackForm.attr('action', 'https://test.smartservice.qld.gov.au/services/submissions/email/feedback/feedback') : '';
+    // if data-recaptcha attribute is not present then insert it
+    !$feedbackForm.attr('data-recaptcha') ? $feedbackForm.attr('data-recaptcha', 'true') : '';
+  };
+  setUrlEnableCaptcha();
+
+  let googleRecaptchaApiKey = checkEnv !== -1
+    ? keys.defGoogleRecaptcha.uat
+    : keys.defGoogleRecaptcha.prod;//This is a v2 key
+  let footerFeedbackGoogleRecaptchaApiKey = checkEnv !== -1
+    ? keys.defFeedbackGoogleRecaptcha.uat
+    : keys.defFeedbackGoogleRecaptcha.prod;//This is a v3 key
+  //v3 Captcha, can have multiples
+  let v3Captcha = (form, greptcha, key, action) => {
+    try {
+      grecaptcha.execute(key, {action: action})
+        .then(function (token) {
+          if (greptcha.length > 0) {
+            if (
+              greptcha.attr('value') !== '' ||
+            greptcha.attr('value').length !== 0 ||
+            greptcha.attr('value') !== undefined) {
+              greptcha.val(token);
+              form.submit();
+              return true;
+            }
+          }
           return false;
-        }
-        grecaptcha.execute();
+        });
+    } catch (e) {
+      return false;
+    }
+  };
+
+  //v2 Captcha, usually is singular
+  let v2Captcha = (form, subBtn, key) => {
+    try {
+      grecaptcha.render(subBtn, {
+        sitekey: key,
+        callback: () => {
+          var response = grecaptcha.getResponse();
+          if (
+            response === '' ||
+              response === undefined ||
+              response.length === 0
+          ) {
+            console.log('Invalid recaptcha');
+            return false;
+          } else {
+            form.submit();
+          }
+        },
       });
+    } catch (e) {
+      grecaptcha.reset();
+      return false;
+    }
+    grecaptcha.execute();
+  };
+
+  let onloadRecaptcha = () => {
+    grecaptcha.ready(function () {
+    // eslint-disable-line
+    //v2 Forms
+      $('form[data-recaptcha="true"]')
+        .find('input[type="submit"], button[type="submit"]')
+        .on('click', e => {
+          e.preventDefault();
+          let subBtn = e.target;
+          let form = $(subBtn).parents('form');
+          var greptcha = form.find('input[name="g-recaptcha-response"]');
+          let manualSitekey = form.attr('data-sitekey');
+          let manualAction = form.attr('data-action');
```

Comment:
data-action should be data-recaptcha-action don't you think or is it ok being data-action, do we have any other locations that this key is used?